pkg/controller/inference/playground_controller.go

```diff
@@ -81,6 +82,24 @@ func (r *PlaygroundReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 
 	logger.V(10).Info("reconcile Playground", "Playground", klog.KObj(playground))
 
+	service := &inferenceapi.Service{}
+	if err := r.Get(ctx, types.NamespacedName{Name: playground.Name, Namespace: playground.Namespace}, service); err == nil {
+		if !controllerutil.HasControllerReference(service) {
+			condition := metav1.Condition{
```

Comment:
We should collect all the condition handlers to one place just like `setPlaygroundCondition` but this can be a follow up when implementing the Pending status for waiting for model creation.

Reply:
ok